src/ui/screens/settings/ViewPreferences.qml

```diff
@@ -67,6 +68,27 @@ MZViewBase {
             onClicked: MZSettings.gleanEnabled = !MZSettings.gleanEnabled
         }
 
+        MZCheckBoxRow {
+                id: localNetwork
+                objectName: "settingLocalNetworkAccess"
+                visible: Qt.platform.os == "ios"
+                anchors {
+                    right: parent.right
+                    left: parent.left
+                    rightMargin: VPNTheme.theme.windowMargin
+                }
+
+                labelText: MZI18n.LocalNetworkAccessLabel
+                subLabelText: MZI18n.LocalNetworkAccessSubLabel
+                isChecked: MZSettings.localNetworkAccess
+                enabled: VPNController.state === VPNController.StateOff
```

Comment:
Should part of the text explain that this is only enabled when the VPN is off? If not, users will show up here with the VPN active and have no indication as to why the setting is disabled.

Reply:
That is a good point. There are other features that depend on the VPN being turned off. I'll check how they handle it.

Reply:
Interesting, we actually don't have this behaviour anywhere in the app. I am pretty sure we used to. I'll ask today during stand up.